### PR TITLE
Enable null safety

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
@brianegan – now that we have our beta out, could we land this?

File: example_async_and_test/lib/main.dart [hunk 1]
```diff
@@ -90,7 +90,7 @@ class CounterHome extends StatelessWidget {
             // `notifyListeners` in the Model).
             ScopedModelDescendant<AbstractModel>(
               builder: (context, child, model) => Text(model.counter.toString(),
-                  style: Theme.of(context).textTheme.headline4),
+                  style: Theme.of(context)!.textTheme.headline4),
```
Comment (brianegan) [inline example_async_and_test/lib/main.dart:93]: Are the `!` necessary for `Theme` examples? Looks like that's non-nullable: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/theme.dart#L119

**Review by brianegan — APPROVED** [review]
Thanks @kevmoo! I'll publish a pre-release with these updates :)

File: example_async_and_test/lib/main.dart (hunk 1 above)
Comment (brianegan) [reply example_async_and_test/lib/main.dart:93]: Just pulled em in, looks like they're safe to remove. Done so on master!

**Comment by lovelessX** [comment]
Any chance we can get a "full release" of the null safety update?

**Pull request merged**